package.json

```diff
@@ -0,0 +1,137 @@
+    "color-name": "^1.1.3",
+    "color-convert": "^1.9.3",
+    "deep-is": "^0.1.3",
+    "define-properties": "^1.1.3",
+    "cli-width": "^3.0.0",
+    "callsites": "^3.1.0",
+    "emoji-regex": "^8.0.0",
+    "es-abstract": "^1.17.6",
+    "error-ex": "^1.3.2",
+    "escape-string-regexp": "^1.0.5",
+    "brace-expansion": "^1.1.11",
+    "es-to-primitive": "^1.2.1",
+    "eslint-visitor-keys": "^1.3.0",
+    "esrecurse": "^4.2.1",
+    "esutils": "^2.0.3",
+    "esprima": "^4.0.1",
+    "external-editor": "^3.1.0",
+    "estraverse": "^4.3.0",
+    "fast-json-stable-stringify": "^2.1.0",
+    "figures": "^3.2.0",
+    "fast-levenshtein": "^2.0.6",
+    "flatted": "^2.0.2",
+    "find-up": "^2.1.0",
+    "fs.realpath": "^1.0.0",
+    "fast-deep-equal": "^3.1.3",
+    "has-symbols": "^1.0.1",
+    "has": "^1.0.3",
+    "graceful-fs": "^4.2.4",
+    "ansi-escapes": "^4.3.1",
+    "has-flag": "^3.0.0",
+    "inherits": "^2.0.4",
+    "iconv-lite": "^0.4.24",
+    "is-arrayish": "^0.2.1",
+    "is-callable": "^1.2.0",
+    "is-date-object": "^1.0.2",
+    "is-glob": "^4.0.1",
+    "is-extglob": "^2.1.1",
+    "is-fullwidth-code-point": "^3.0.0",
+    "is-string": "^1.0.5",
+    "isexe": "^2.0.0",
+    "is-regex": "^1.1.0",
+    "isarray": "^1.0.0",
+    "json-schema-traverse": "^0.4.1",
+    "js-tokens": "^4.0.0",
+    "locate-path": "^2.0.0",
+    "lodash": "^4.17.19",
+    "load-json-file": "^2.0.0",
+    "minimist": "^1.2.5",
+    "levn": "^0.3.0",
+    "mimic-fn": "^2.1.0",
+    "mkdirp": "^0.5.5",
+    "minimatch": "^3.0.4",
+    "ms": "^2.1.2",
+    "nice-try": "^1.0.5",
+    "mute-stream": "^0.0.8",
+    "object-inspect": "^1.8.0",
+    "object-keys": "^1.1.1",
+    "os-tmpdir": "^1.0.2",
+    "object.assign": "^4.1.0",
+    "onetime": "^5.1.0",
+    "p-try": "^1.0.0",
+    "p-locate": "^2.0.0",
+    "p-limit": "^1.3.0",
+    "path-is-absolute": "^1.0.1",
+    "parent-module": "^1.0.1",
+    "parse-json": "^2.2.0",
+    "path-key": "^2.0.1",
+    "path-exists": "^3.0.0",
+    "pify": "^2.3.0",
+    "path-parse": "^1.0.6",
+    "punycode": "^2.1.1",
+    "is-symbol": "^1.0.3",
+    "path-type": "^2.0.0",
+    "prelude-ls": "^1.1.2",
+    "pkg-dir": "^2.0.0",
+    "resolve": "^1.17.0",
+    "run-async": "^2.4.1",
+    "restore-cursor": "^3.1.0",
+    "resolve-from": "^4.0.0",
+    "safer-buffer": "^2.1.2",
+    "rxjs": "^6.6.0",
+    "shebang-command": "^1.2.0",
+    "shebang-regex": "^1.0.0",
+    "sprintf-js": "^1.0.3",
+    "signal-exit": "^3.0.3",
+    "slice-ansi": "^2.1.0",
+    "string.prototype.trimend": "^1.0.1",
+    "string-width": "^4.2.0",
+    "strip-ansi": "^5.2.0",
+    "strip-bom": "^3.0.0",
+    "string.prototype.trimstart": "^1.0.1",
+    "supports-color": "^5.5.0",
+    "tmp": "^0.0.33",
+    "through": "^2.3.8",
+    "type-check": "^0.3.2",
+    "tslib": "^1.13.0",
+    "uri-js": "^4.2.2",
+    "type-fest": "^0.8.1",
+    "cli-cursor": "^3.1.0",
+    "function-bind": "^1.1.1",
+    "argparse": "^1.0.10"
```

Comment:
I couldn't help noticing that your package json file is has a lot installed dependencies that are not being used by your app. Can I ask why those dependencies were added :thinking: ?

Reply:
Hey @Torres-ssf, we initially tried to install bootstrap toggle package and later removed it. I just noticed these. I have updated the same. Thank you for pointing it out. 
Good day 👍